<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaikoTool</class>
 <widget class="QMainWindow" name="TaikoTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1248</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>TaikoTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="songSelectTab">
       <attribute name="title">
        <string>All Songs</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="songTabSplitter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeWidget" name="songTree">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Source</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Title</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Genre</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Order</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>UniqueId</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Modified</string>
            </property>
           </column>
          </widget>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="sizeIncrement">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="songNameJPLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>75</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Select a song</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="songNameENLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="songSubLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="difficultiesLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>150</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="textFormat">
               <enum>Qt::RichText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="buildButtonWidget" native="true">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QPushButton" name="addToBuildButton">
                 <property name="text">
                  <string>Add to Build</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="removeFromBuildButton">
                 <property name="text">
                  <string>Remove from Build</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="songPropertiesStackedWidget">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>225</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="singleSongPage">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QWidget" name="singleSongPropertyWidget" native="true">
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="3" column="0">
                    <widget class="QCheckBox" name="songDebugCheckbox">
                     <property name="text">
                      <string>Debug</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">singleSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item row="0" column="0">
                    <widget class="QCheckBox" name="songSecretCheckbox">
                     <property name="text">
                      <string>Secret (Unlockable?)</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">singleSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QCheckBox" name="songDLCCheckbox">
                     <property name="text">
                      <string>DLC</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">singleSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item row="4" column="0">
                    <widget class="QCheckBox" name="songRecordableCheckbox">
                     <property name="text">
                      <string>Recordable</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">singleSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="multiSongPage">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QWidget" name="multiSongProperyWidget" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>107</width>
                    <height>225</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QPushButton" name="setAllSecretButton">
                     <property name="text">
                      <string>All Secret</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setAllDLCButton">
                     <property name="text">
                      <string>All DLC</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setAllDebugButton">
                     <property name="text">
                      <string>All Debug</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setAllRecordableButton">
                     <property name="text">
                      <string>All Recordable</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setNoneSecretButton">
                     <property name="text">
                      <string>None Secret</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setNoneDLCButton">
                     <property name="text">
                      <string>None DLC</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setNoneDebugButton">
                     <property name="text">
                      <string>None Debug</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="setNoneRecordableButton">
                     <property name="text">
                      <string>None Recordable</string>
                     </property>
                     <attribute name="buttonGroup">
                      <string notr="true">multiSongPropButtonGroup</string>
                     </attribute>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="playStopButton">
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="focusPolicy">
               <enum>Qt::TabFocus</enum>
              </property>
              <property name="toolTip">
               <string>Preview song</string>
              </property>
              <property name="text">
               <string>▶</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="buildTab">
       <attribute name="title">
        <string>Build</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QSplitter" name="buildTabSplitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="buildList"/>
          <widget class="QFrame" name="frame_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>250</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLabel" name="buildReadyLabel">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <weight>50</weight>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>Add songs to the build in the &quot;All Songs&quot; tab</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QWidget" name="widget_3" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QCheckBox" name="showVersionMSCheckbox">
                 <property name="text">
                  <string>Show Taiko Tool version string in main menu</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showVersionSSCheckbox">
                 <property name="text">
                  <string>Show Taiko Tool version string in song select menu</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showSourceCheckbox">
                 <property name="text">
                  <string>Show song source in song sub-text</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buildButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Build...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1248</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTaiko_Tool">
    <property name="title">
     <string>Taiko Tool</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Load...</string>
     </property>
     <addaction name="actionBase_game"/>
     <addaction name="actionPS4Game"/>
     <addaction name="actionWii_U_game"/>
    </widget>
    <addaction name="menuLoad"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuTaiko_Tool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionBase_game">
   <property name="text">
    <string>Base game</string>
   </property>
  </action>
  <action name="actionPS4Game">
   <property name="text">
    <string>PS4 Game</string>
   </property>
  </action>
  <action name="actionWii_U_game">
   <property name="text">
    <string>Wii U game</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="multiSongPropButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
  <buttongroup name="singleSongPropButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
